lib/moment-dist-moment.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module 'moment/dist/moment.js' {
+    import moment = require('moment');
+    export = moment;
+}
```

Comment:
@susnux Not sure about it. Is there a better way to provide typing for a specific file?

Reply:
No I think this is the best way to do, but it looks weird, I think it should be:

```ts
declare module 'moment/dist/moment.js' {
    import moment from 'moment'
    export = moment
}
```

Reply:
but why cant we just import `'moment'` without any path?

Reply:
> No I think this is the best way to do, but it looks weird, I think it should be:

Updated.